src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2Engine.java

```diff
@@ -211,13 +203,12 @@ public void writeHeader(final VariantContextWriter vcfWriter, final SAMSequenceD
 
     private Set<VCFHeaderLine> getSampleHeaderLines(){
         final Set<VCFHeaderLine> sampleLines = new HashSet<>();
-        if (printTCGAsampleHeader) {
-            //NOTE: This will only list the first bam file for each tumor/normal sample if there is more than one
-            final Map<String, String> normalSampleHeaderAttributes = ImmutableMap.of("ID", "NORMAL", "SampleName", MTAC.normalSampleName);
-            final Map<String, String> tumorSampleHeaderAttributes = ImmutableMap.of("ID", "TUMOR", "SampleName", MTAC.tumorSampleName);
-            sampleLines.add(new VCFSimpleHeaderLine("SAMPLE", normalSampleHeaderAttributes));
-            sampleLines.add(new VCFSimpleHeaderLine("SAMPLE", tumorSampleHeaderAttributes));
+        if (hasNormal()) {
+            final Map<String, String> normalSampleHeaderAttributes = ImmutableMap.of("ID", "NORMAL", "Sample", MTAC.normalSampleName);
```

Comment:
`Sample` as a subfield of `sample` header line feels redundant. Maybe `name` or `sampleName`?